package.json

```diff
@@ -159,7 +159,7 @@
     "settings": {
       "import/resolver": {
         "webpack": {
-          "config": "./internals/webpack/webpack.dev.babel.js"
+          "config": "./internals/webpack/webpack.prod.babel.js"
```

Comment:
@samit4me Yes, unfortunately #webpackmagic seems to be a an anti-pattern for many attempting to add any sizeable library to RBP, based on my recollection of issue history. I have not studied the issue and claim no expertise or special understanding, but RBP's webpack configuration feels "over engineered" to me. That's likely to be due to my lack of understanding. Still, the situation as described above, seems odd.

It would be nice to have studied input from a qualified webpack wizard.